_studio/mfx_lib/encode_hw/CMakeLists.txt

```diff
@@ -58,6 +69,12 @@ foreach( prefix ${MSDK_LIB_ROOT}/shared/src )
     )
 endforeach()
 
+foreach( dir ${hevcdir} )
+  file( GLOB_RECURSE srcs "${dir}/*.cpp" )
```

Comment:
I think @onabiull  is moving towards explicit list of files rather than wildcards. I suggest to sync with him

Reply:
wildcards removed